src/app/core/callendar/model/resource-list.ts

```diff
@@ -0,0 +1,16 @@
+export class ResourceList {
+
+    private _list: { 'id', 'title' }[] = [];
```

Comment:
Maybe define a Resource model in a separate file instead of using '{ 'id', 'title' }' ?